### DISCUSSION TITLE
Accessing my newly hacked WUUK

### DISCUSSION THREAD

**Discussion by SamBozman** [themactep/thingino-firmware#187] [General]
Hi everyone,
I have hacked the WUUK camera and it was successful. Now I wanted to view the output, so I went to the 'Getting Started' page, but I can not find an example of what I need to type in the address bar in order to view the limited frame rate output from my camera.
I have a name and password for this device. I suspect I need to type this name and password and maybe the port?(80) in some format in order to access the output. ??

Can anyone give me some suggestions?

Thanks
Sam

**Comment by gtxaspec** [comment]
hi @SamBozman !
the webui is located at `http://ip`, example `http://192.168.1.20`
i'll add the url to the wiki, thank you! let us know if you have more questions.

**Reply by SamBozman** [reply]
Thank you, that did work. I gave the device a Hostname of "WUUK1" and a root password of "hacker99WUUK1".
When I loaded the IP address, it asked me for a username and password. I tried the Hostname in place of the username and the root password in place of the password, but that did not work. Can you point me in the right direction?

Thanks
Sam

**Reply by wltechblog** [reply]
The webui login is username root and the password you set in the portal.

**Reply by SamBozman** [reply]
Thank you again. Yes, I am in. Nice interface, and it looks like it has a lot of features. This is very interesting as I am tired of my camera's going out of service.
 I hope we can continue with the development of Thingino :)

**Reply by gtxaspec** [reply]
i've updated the [getting started](https://github.com/themactep/thingino-firmware/wiki/Getting-Started) wiki article to be a little clearer and provide more direction for new users regarding the webui, links and addresses =D

**Reply by SamBozman** [reply]
Thanks. I find the documentation quite confusing, but I guess I will eventually grasp the concepts. 
Thank you for your help.
